topics/proteomics/tutorials/metaquantome-data-creation/tutorial.md

```diff
@@ -0,0 +1,656 @@
+---
+layout: tutorial_hands_on
+
+title: metaQuantome Data creation tutorial
+zenodo_link: "https://doi.org/10.5281/zenodo.4037137"
+questions:
+ - "How do I perform functional and taxonomy analysis on metaproteomics data?"
+ - "How can I perform quantitation on metaproteomics data?"
+ - "How do I create inputs that can be used in metaquantome to examine differentially expressed proteins?"
+objectives:
+  - "A taxonomy, functional and quantitational analysis of metaproteomic mass spectrometry data."
+time_estimation: "1h"
+key_points:
+  - "Use dataset collections"
+  - "With SearchGUI and PeptideShaker you can gain access to multiple search engines"
+  - "Learning the basics of SQL queries can pay off"
+contributors:
+  - subinamehta
+  - timothygriffin
+  - pratikdjagtap 
+  - emmaleith
+  - mariecrane
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+Metaproteomics involves characterization of community level expression of microbial proteins from an environmental 
+or clinical sample. Metaproteomics data is primarily used to determine the functional status of the microbiome under 
+study along with its taxonomic composition. The Galaxy-P team published a software suite named [metaQuantome](https://www.mcponline.org/content/18/8_suppl_1/S82) to enable quantitative and statistical analysis and visualization of functional, taxonomic expression as well as functional and 
+taxonomy interaction. metaQuantome leverages peptide level quantitative information to analyze the taxonomic, functional 
+expression within the microbial community in different conditions.
+
+<p align="center">
+  <img width="850" height="500" src="../../images/microbiome.png">
+</p>
```

Comment:
The linter seems to have issues with how the images are being called out after converting the tutorial.md to html:
```
- ./_site/training-material/topics/proteomics/tutorials/metaquantome-data-creation/tutorial.html
  *  image ../../images/microbiome.png does not have an alt attribute (line 444)
...
```

In other tutorials, images are called out using the following notation:
```
![CEL-Seq2 Scheme](../../images/celseq2_schema.svg "Read1 encapsulates the barcodes, and Read2 the mRNA sequence")
```

where `CEL-Seq2 Scheme` is the alt_text and `"Read1 encapsulates the barcodes, and Read2 the mRNA sequence"` is inserted as the legend for the figure. 

I suggest either:
1. Replacing html-style image call-outs to the markdown style shown above OR
1. Updating existing html-style image call-outs to include `alt` and `title` attributes, like:

```
<p align="center">
   <img width="850" height="500" src="../../images/microbiome.png" alt="<alt text>" title="<title>" />
   </p>
```

Maybe one of these options is preferred over the other by the training team?

Reply:
thanks a lot for reviewing @malloryfreeberg! 

@subinamehta please use the markdown formatting for including slides, not html. If you need to modify the width of the image, you can do this like: `![description](path/to/image.png){: width="75%"}`